### DISCUSSION TITLE
[UX] Considering Turntable as default Orbit style

### DISCUSSION THREAD

**Discussion by inkapnea** [FreeCAD/FreeCAD#16395] [Discussions that will turn in to issues but need refining]
Please consider setting Turntable the default orbit style in FreeCAD for 1.0.

From my personal experience I can say, the first time I launched FreeCAD and started to familiarize with the interface, I thought the software was broken.
As soon I tried to orbit the view with the mouse, the view was tumbling awkwardly out of my control. It was frustrating not being able to point the Z axis straight up with the mouse, just where it was before I touched the mouse.
So my first impression was that FreeCAD was badly broken, or poorly made software, inferior to other 3D programs I used before.
Had to google for a solution how to tame that awkward behaving CAD. But in the meantime my excitement about learning this new (for me) FLOSS software dropped. Because it drops when some basic things turn out to be overcomplicated, and you haven't started yet.

I know, this is only my subjective view, but I think this is not an uncommon experience for new users to this software. And I'm confident the Trackball Orbit ruins the user experience in the first minute such a user launches FreeCAD for the first time.
In that moment I was already contemplating about giving FreeCAD a pass, and jumping to the next CAD from the list of CAD packages to try.

This was my subjective experience, and is worth little as much.
But the rationale behind my demand at making the Turntable Orbit default is:

1)
It's the default in other 3D software packages, so users are accustomed to - Blender, Tinkercad, SketchUp, Onshape, Plasticity, Slicer and Cura. I can't speak for Fusion360, Solidworks, AutoCAD and others because I never used them.

2)
We as humans naturally tend to keep the horizon horizontal.
When we explore an object that is static on the ground, we tend to walk around it. Get closer and further, tilt our vision up and down. Those are our 3 basics movements. Rolling our point of view comes later, only if the object is not clearly visible because of a cavity, shallow light conditions, an oblique writing...

3)
The seasoned user is familiar with the Preferences dialog and knows how to set the software as he wishes. The new user doesn't know where and what to search for.

Let's not ruin the experience for new users coming to FreeCAD. It's a trivial thing changing the orbit type, but it could make a difference in creating a positive first impression (or avoiding a bad first impression).

Are there some reasons why Trackball is the default? I can't think of none. Maybe some historical reasons?

After the release of 1.0 there will be a surge of new users trying FreeCAD for the first time. Let's make that experience pleasurable.

**Comment by luzpaz** [comment]
This is too involved for github discuss. Should be put on the forum.freecad.org for more exposure.

**Comment by inkapnea** [comment]
OK. I created a new topic on the forum.
https://forum.freecad.org/viewtopic.php?t=90491
